fog/view/server/src/sharding_strategy.rs

```diff
@@ -95,11 +95,18 @@ impl EpochShardingStrategy {
             return true;
         }
 
-        let epoch_block_range_length =
-            self.epoch_block_range.end_block - self.epoch_block_range.start_block;
-        let minimum_processed_block_count = epoch_block_range_length / 2;
-
-        u64::from(processed_block_count) >= minimum_processed_block_count
+        // Original logic requires/assumes a 50% overlap.
+        // This doesn't really work if the we try a minimal overlap to optimize
+        // for cost. We should revisit at some point and create
+        // alternative strategies. This logic should drive the health
+        // check endpoint that k8s can consume more than a response to
+        // the router.
+        // let epoch_block_range_length =
+        //     self.epoch_block_range.end_block -
+        // self.epoch_block_range.start_block;
+        // let minimum_processed_block_count = epoch_block_range_length / 2;
+
+        u64::from(processed_block_count) >= 1
```

Comment:
:question: Should a similar change be made in https://github.com/mobilecoinfoundation/mobilecoin/blob/master/fog/ledger/server/src/sharding_strategy.rs ?

Reply:
I'll fix that.  

For some reason ledger seems to work, but why would that be the case? 

Maybe because fog-test-client is only hitting the parts of ledger that are integrated with the router service? 

I don't think that's true because I do see cpu load on the stores.